src/components/plots/xyPlot.ts

```diff
@@ -149,22 +149,22 @@ module Plottable {
     }
 
     /**
-     * Gets the automatic domain adjustment setting for visible points.
+     * Gets the automatic domain adjustment mode for visible points.
      */
-    public autorange(): string;
+    public autorangeMode(): string;
     /**
-     * Sets the automatic domain adjustment for visible points to operate against the X Scale, Y Scale, or neither.
+     * Sets the automatic domain adjustment mode for visible points to operate against the X Scale, Y Scale, or neither.
      * If "x" or "y" is specified the adjustment is immediately performed.
      *
-     * @param {string} scaleName One of "x"/"y"/"none".
+     * @param {string} mode One of "x"/"y"/"none".
      *   "x" will adjust the x Scale in relation to changes in the y domain.
      *   "y" will adjust the y Scale in relation to changes in the x domain.
      *   "none" means neither Scale will change automatically.
      * @returns {XYPlot} The calling XYPlot.
      */
-    public autorange(scaleName: string): XYPlot<X, Y>;
-    public autorange(scaleName?: string): any {
-      if (scaleName == null) {
+    public autorangeMode(mode: string): XYPlot<X, Y>;
```

Comment:
Why not keep the parameter name same as the method name? We tend to do that in the rest of our codebase